### DISCUSSION TITLE
求打包instagram，WIN10版本

### DISCUSSION THREAD

**Discussion by halavah** [tw93/Pake#231] [Q&A]
![image](https://user-images.githubusercontent.com/55541070/211121682-0660143d-dffe-4d4f-b0f6-8a0f45f832a4.png)

**Comment by Tlntin** [comment]
看了一下你的git actions日志。缺对应的windows/linux图标，所以只打包了mac版的。
解决办法，运行"icns2png.py"文件，即可生成ico png图标。

**Reply by tw93** [reply]
这里我们要不要做一个兜底，直接没有图标用默认图标即可，已经有不少同学碰到这个问题啦

**Reply by heiyuan0801** [reply]
这个想法不错 避免这个问题出现

**Reply by halavah** [reply]
已解决。昨天太困了，没仔细看文档。

推荐用 https://icon-icons.com/zh/ ，生成 .icns、.ico、.png图标

**Comment by halavah** [comment]
## 1.简单配置
 - ![image](https://user-images.githubusercontent.com/55541070/211131702-6ac1b464-85bf-46e8-89f3-69044ef03995.png)

## 2.推荐【icon】生成网址
- https://icon-icons.com/zh/%E5%9B%BE%E6%A0%87/instagram-%E6%96%B0-%E6%A0%87%E5%BF%97/66804
-![image](https://user-images.githubusercontent.com/55541070/211131665-f14d459c-4eea-4782-81cf-dc0c2a3f997f.png)

## 3.完美效果
- ![image](https://user-images.githubusercontent.com/55541070/211131755-4679a358-a665-4ead-a924-d32848adaa45.png)

**Comment by aqpmzngldh** [comment]
我这个不会生成msi，只会生成5个缺这个
